### ISSUE TITLE
Multiple species example in README needs `--tree` at the last step

### ISSUE THREAD

**Issue by mptrsen** [DessimozLab/read2tree#3]
The documentation needs `--tree` for the multi-species mode in the last step to actually produce the tree.

**What happens:**

The last step is executed, but no tree is produced (IQTree is never run)

**Expected:**

Tree.

**Solution:**

```
read2tree --standalone_path marker_genes/ --ouput_path output --merge_all_mappings --tree
```

**Comment by smajidian** [comment]
Dear Malte Petersen
Thanks for using our tool. 
I just noticed a type in the tutorial; please make sure the spelling of  `--output_path`. When I solved this, there wasn't other error in my run. So, I'm not sure I could reproduce your case. 
I'm wondering whether you tried the test case ?
It would be great if we can have access to the log file `mplog.log`. You may put its text here. Alternatively, you can email us at sina.majidian@unil.ch
thanks.

**Comment by mptrsen** [comment]
Yes, you're right about the typo that used to be `--ouput_path`. However, that wasn't the issue (I had fixed it in my command already), but it should also be corrected (edited the issue text).

The point is that Read2Tree will only produce a tree when called with the `--tree` option. Without it, Read2Tree will not produce a tree. So I think it should be included in the example.

**Comment by smajidian** [comment]
That's right. I got it. Thank a lot for pointing that out. I've just updated the documentation. 
Best regards,
Sina
